tests/lambda_endpoints/conftest.py

```diff
@@ -77,17 +84,29 @@ def events_client():
         yield boto3.client("events", region_name="us-west-2")
 
 
-# NOTE: This test_engine scope is function.
-# With this scope, all the changes to the database
-# is only visible in each test function. It gets
-# cleaned up after each function.
+# NOTE: The default scope is function, so each test function will
+#       get a new database session and start fresh each time.
 @pytest.fixture()
-def test_engine():
-    """Create an in-memory SQLite database engine."""
-    with patch.object(db, "get_engine") as mock_engine:
-        engine = create_engine("sqlite:///:memory:")
-        mock_engine.return_value = engine
+def session():
+    """Create a test postgres database engine."""
+    from sds_data_manager.lambda_code.SDSCode.database import database as db
+
+    with patch.object(db, "Session") as mock_session:
+        connection = "sqlite:///:memory:"
```

Comment:
Does this setup postgres database engine?

Reply:
No, that is happening in the subsequent PRs. This is still only sqlite.